modules/fileout_netcdf/FONcArrayStructure.h

```diff
@@ -0,0 +1,66 @@
+// FONcArrayStructure.h
+
+// This file is part of BES Netcdf File Out Module
+
+// // Copyright (c) The HDF Group, Inc. and OPeNDAP, Inc.
+//
+// This is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License as published by the Free
+// Software Foundation; either version 2.1 of the License, or (at your
+// option) any later version.
+//
+// This software is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+// or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public
+// License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+//
+// You can contact OPeNDAP, Inc. at PO Box 112, Saunderstown, RI. 02874-0112.
+// You can contact The HDF Group, Inc. at 410 E University Ave,
+// Suite 200, Champaign, IL 61820
+
+
+#ifndef FONcArrayStructure_h_
+#define FONcArrayStructure_h_ 1
+
+#include <libdap/Structure.h>
+#include "FONcBaseType.h"
+#include "FONcArrayStructureField.h"
```

Comment:
Since FONcArrayStructureField is used as a pointer here, the header does not don't need to be included. You can replace the include with a forward declaration like:

class FONcArrayStructureField;